parl/framework/algorithm.py

```diff
@@ -176,7 +176,7 @@ def get_behavior_model(self):
         """
         return self.model
 
-    def predict(self, inputs, states):
+    def _predict(self, inputs, states):
```

Comment:
Why do we need to change predict to _predict?

Reply:
Because for some algorithms like SimpleQ, they need to insert some preprocessing (e.g., exploration_rate) before doing the prediction(). So they can override the predict() function but inside that they can still call _predict()